packages/playground/src/components/LogMessage.vue

```diff
@@ -80,16 +80,7 @@ export default {
     const props = toRef(_props);
 
     const collapsable = computed(() => props.value.log.data.message?.length > 200);
-    const collapsed = ref(true);
-
-    watch(
-      () => props.value.log.id,
-      () => {
-        if (!collapsed.value) {
-          collapsed.value = true;
-        }
-      },
-    );
+    const collapsed = computed(() => !!props.value.log?.id);
```

Comment:
While testing from a private window, I got many errors, and when I clicked on show more I got  
![image](https://github.com/user-attachments/assets/36715045-64a2-4d15-b6f9-89a9f0c16ace)

![image](https://github.com/user-attachments/assets/e143dccc-7948-4539-bb93-abdcb57608a7)

Reply:
[Screencast from 18-05-25 16:50:30.webm](https://github.com/user-attachments/assets/3c582250-30a9-42ec-9483-1ec15d869f4d)